package.json

```diff
@@ -1,19 +1,41 @@
 {
   "name": "svelte-app",
   "version": "1.0.0",
+  "jest": {
+    "transform": {
+      "^.+\\.svelte$": "svelte-jester"
+    },
+    "moduleFileExtensions": [
+      "js",
+      "svelte"
+    ],
+    "setupFilesAfterEnv": [
+      "@testing-library/jest-dom/extend-expect"
+    ]
+  },
   "scripts": {
     "build": "rollup -c",
     "dev": "rollup -c -w",
-    "start": "sirv public --single"
+    "start": "sirv public --single",
+    "test": "jest src",
+    "test:watch": "npm run test -- --watch"
```

Comment:
Did you have it auto running tests every time you saved?

Reply:
I wasn't initially but I have tested it and it works to auto run it. I just had to increase the number of watchers.